### ISSUE TITLE
Modify menu item

### ISSUE THREAD

**Issue by kaiserdj** [sindresorhus/electron-context-menu#134]
Hi,
I would like to know if there is any option to be able to modify the menu after being created, obtain the menu or inject new MenuItem

**Comment by sindresorhus** [comment]
Yeah, I can see how that could be useful. Currently, there's only a way to append/prepend to the default menu. Maybe we could add a `updateMenu` option that is called before the menu is shown.

**Comment by kaiserdj** [comment]
It seems like a very good solution

**Comment by sindresorhus** [comment]
Fixed in `9032711`. Adds an `updateMenu` option, called with the final menu template just before the menu is shown. You can filter, reorder, or inject items.
